README.rst

```diff
@@ -81,4 +81,26 @@ Windows
     pip install virtualenv
     virtualenv <your-env>
     <your-env>\Scripts\activate
-    <your-env>\Scripts\pip.exe install bigquery-magics
\ No newline at end of file
+    <your-env>\Scripts\pip.exe install bigquery-magics
+
+Example Usage
+-------------
+
+To use these magics, you must first register them. Run the ``%load_ext bigquery-magic``
```

Comment:
I believe this needs to match the import name, not the package name.

```suggestion
To use these magics, you must first register them. Run the ``%load_ext bigquery_magics``
```

See: https://github.com/googleapis/python-bigquery-magics/blob/23d54cd6721e362974f0119534fad726dc9a9608/samples/snippets/query_params_scalars.py#L27

Reply:
Done.